src/satctl.c

```diff
@@ -90,7 +118,7 @@ int main(int argc, char **argv)
 	int csp_version = 2;
 	char * rtable = NULL;
 	char * tun_conf_str = NULL;
-	char * csp_zmqhub_addr[10];
+	char * csp_zmqhub_addr[20];
```

Comment:
I think 20 is a bit low - maybe just do 128 (you can throw a fully qualified hostname in there).